codesandbox/codesandbox-client
# feat(dashboard): update repo cards

What to test for:
* regular repo cards
* private repos should have the icon next to the number of branches
* view only now shows restricted
* import repo button is aligned with the cards and disabled when limits are reached
* list view is also updated
<img width="1534" alt="Screenshot 2022-11-29 at 17 31 34" src="https://user-images.githubusercontent.com/9945366/204588731-29562afa-e5d3-48e7-9012-3d4f985bfaa2.png">

<img width="1792" alt="Screenshot 2022-11-29 at 17 29 53" src="https://user-images.githubusercontent.com/9945366/204588700-f1132fb7-c85d-4302-9267-b5b946af8790.png">
<img width="1510" alt="Screenshot 2022-11-29 at 17 30 45" src="https://user-images.githubusercontent.com/9945366/204588713-bbdcda71-06a5-47e9-99e3-7252f81dc6c6.png">
<img width="1528" alt="Screenshot 2022-11-29 at 17 31 27" src="https://user-images.githubusercontent.com/9945366/204588723-b2f34d68-37ab-4e66-9497-5775e9ecd359.png">
